Pipfile

```diff
@@ -30,10 +30,13 @@ jsonschema = "*"
 numpy = "*"
 psycopg2-binary = "*"
 requests = "*"
-sqlalchemy-utils = "*"
+# sqlalchemy-utils 0.36.8 introduced a bug
+# Can probably reset this to * once the next patch comes out
+sqlalchemy-utils = "==0.36.7"
```

Comment:
let's file a ticket so we remember to check on this in a few weeks.

Reply:
@ben yep - this is a simplification @jonahkagan came up with per this thread: https://votingworks.slack.com/archives/CKCVA0F9S/p1593121185202400?thread_ts=1593119993.195400&cid=CKCVA0F9S